docs/content/preview/api/ysql/user-defined-subprograms-and-anon-blocks/language-plpgsql-subprograms/plpgsql-syntax-and-semantics/executable-section/compound-statements/_index.md

```diff
@@ -17,9 +17,9 @@ The following table lists all of the [compound PL/pgSQL executable statements](.
 - The _Statement Name_ column links to the page where the semantics are described.
 - The _Syntax rule name_ column links to the definition on the omnibus [Grammar Diagrams](../../../../../syntax_resources/grammar_diagrams/) reference page.
 
-| STATEMENT NAME                                | SYNTAX RULE NAME                                                                           | COMMENT                                                                                                              |
-| --------------------------------------------- | ------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------- |
-| [block statement](../../#plpgsql-block-stmt)  | [plpgsql_block_stmt](../../../../../syntax_resources/grammar_diagrams/#plpgsql-block-stmt) |                                                                                                                      |
-| [loop statement](./loop-exit-continue/)       | [plpgsql_loop_stmt](../../../../../syntax_resources/grammar_diagrams/#plpgsql-loop-stmt)   | [unbounded loop](./loop-exit-continue/infinite-and-while-loops/), [bounded loop](./loop-exit-continue/#bounded-loop) |
-| [if statement](./if-statement/)               | [plpgsql_if_stmt](../../../../../syntax_resources/grammar_diagrams/#plpgsql-if-stmt)       |                                                                                                                      |
-| [case statement](./case-statement/)           | [plpgsql_case_stmt](../../../../../syntax_resources/grammar_diagrams/#plpgsql-case-stmt)   |                                                                                                                      |
+| STATEMENT NAME                                | SYNTAX RULE NAME                                                                           | COMMENT                                                                                                                             |
```

Comment:
order of items in the table - should it match order of pages

Reply:
No, these orders cannot, in general, match[\*]. Each of the two _\_index.md_ pages for, respectively, _basic-statements_ and _compound-statements_, lists each statement by name. Alphabetical ordering by name helps the human scan the page by eye to find the statement of interest. There’s a choice between ordering by the names in the first column (“statement name”) or the second column (“syntax rule name”). I had ordered by the second column. I see now that this was a bad choice—not least ‘cos every syntax rule name starts the same with _plpgsql\__. (And the names are a bit obscure.) So I reversed my decision and now I’ve ordered by the first column. This is better because the statement names are more conversational.

[\*] The orders can’t match because, in some cases, two or more rules are described on the same referenced page—and there, they are ordered in a pedagogically natural way.